### DISCUSSION TITLE
Docker tags

### DISCUSSION THREAD

**Discussion by Jon-NJ** [advplyr/audiobookshelf#175] [Ideas]
With other projects I have sometimes had difficulty determining which version of the software is installed when it was pulled with the "latest" tag. This is often a consideration when updating. Because of this I now always try to pull a tagged release. I'm sure it is early days still, and I don't know how much trouble this is with the current workflow. But it should not be that difficult to also add a version tag.

**Comment by advplyr** [comment]
Pushing a version tag with the latest tag won't be a problem. It makes sense now that the project has become more stable.

**Reply by advplyr** [reply]
Just pushed the first docker image with version tag `v1.6.13`. No problems
